frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseProjectDependentAction.java

```diff
@@ -21,6 +21,15 @@
 
 @ExtensionMethod({GitMacheteBundle.class})
 public abstract class BaseProjectDependentAction extends DumbAwareAction implements IWithLogger {
+
+  // Let's eagerly load these classes so that they do NOT end up loaded from an action `update` method.
```

Comment:
Loading of classes in JVM turned out even more dynamic than what I've expected 😅